driverthroughput/main.py

```diff
@@ -53,41 +57,57 @@ def throughput_standard(self,
         :type iterations: int
         :param tensor_bytes: list of tensor sizes in bytes
         :type tensor_bytes: list[int], optional
+        :param languages: which language to use for the tester "cpp" or "fortran"
+        :type languages: str
+        :param wall_time: allotted time for database launcher to run
+        :type wall_time: str
         """
         logger.info("Starting throughput scaling tests")
         logger.info(f"Running with database backend: {get_db_backend()}")
         logger.info(f"Running with launcher: {launcher}")
 
-        exp = create_folder(exp_name, launcher)
-
-        for db_node_count in db_nodes:
-
+        exp, result_path = create_folder(exp_name, launcher)
+        write_run_config(result_path,
+                    colocated=0,
+                    client_per_node=clients_per_node,
+                    client_nodes=client_nodes,
+                    database_nodes=db_nodes,
+                    database_cpus=db_cpus,
+                    iterations=iterations,
+                    tensor_bytes=tensor_bytes,
+                    language=languages,
+                    wall_time=wall_time)
+        permss = list(product(db_nodes, db_cpus))
+        for permsss in permss:
+            dbn, dbc = permsss
             # start the database only once per value in db_nodes so all permutations
             # are executed with the same database size without bringin down the database
             db = start_database(exp,
                                 db_node_feature,
                                 db_port,
-                                db_node_count,
-                                db_cpus,
+                                dbn,
+                                dbc,
                                 None, # not setting threads per queue in throughput tests
```

Comment:
Switching to less descriptive variable names will make it harder for the next person to maintain this code

Reply:
I agree, I will change